src/test/java/plugins/AnalysisCollectorPluginTest.java

```diff
@@ -329,6 +330,33 @@ public void should_aggregate_warnings_in_dashboard_portlet() {
         assertThat(dashboard, not(hasWarningsFor(job, TASKS, TASKS_ALL)));
     }
 
+    @Test
+    @WithPlugins({"git@2.4.0", "workflow-aggregator@1.10"})
+    public void should_compute_annotations_on_workflow() {
+        MavenInstallation.installMaven(jenkins, "M3", "3.3.3");
+        WorkflowJob job = jenkins.jobs.create(WorkflowJob.class);
+        job.script.set(
+            "node {\n" +
+               job.copyResourceStep("/analysis_collector_plugin/checkstyle-result.xml") +
+               job.copyResourceStep("/analysis_collector_plugin/findbugs.xml") +
+            "  step([$class: 'FindBugsPublisher', pattern: '**/findbugs.xml'])\n" +
+            "  step([$class: 'CheckStylePublisher'])\n" +
+            "  step([$class: 'AnalysisPublisher'])\n" +
+            "}");
+        job.sandbox.check();
+        job.save();
+        final Build build = job.startBuild();
+        build.shouldSucceed();
+
+        assertThat(build, hasAction("Static Analysis Warnings"));
+
+        AnalysisCollectorAction action = new AnalysisCollectorAction(build);
+        action.open();
+
+        assertThat(action.getNumberOfWarnings(), is(FINDBUGS_ALL + CHECKSTYLE_ALL));
+        assertThat(action.getNumberOfNewWarnings(), is(FINDBUGS_ALL + CHECKSTYLE_ALL));
+    }
```

Comment:
True, though in this case I think what is really being asserted is that the build does not explode in a fireball. :-)